config/webpack.config.js

```diff
@@ -21,6 +21,8 @@ const {
   getWebpackEntryPoints,
 } = require('./manifest-helpers');
 
+const vaMedalliaStylesFilename = 'va-medallia-styles';
```

Comment:
```suggestion
// TODO: refactor the other approach for creating files without the hash so that we're only doing that in the webpack config: https://github.com/department-of-veterans-affairs/vets-website/blob/a012bad17e5bf024b0ea7326a72ae6a737e349ec/src/site/stages/build/plugins/process-entry-names.js#L35
const vaMedalliaStylesFilename = 'va-medallia-styles';
```